tests/functional/ImportExportBaseTest.php

```diff
@@ -14,6 +14,7 @@
 abstract class ImportExportBaseTest extends TestCase
 {
     public const TABLE_OUT_CSV_2COLS = 'out_csv_2Cols';
+    public const TABLE_OUT_CSV_2COLS_NO_INSERT = 'out_csv_2Cols_no_insert';
```

Comment:
na co potrebujeme toto?

Reply:
a k tomu tests/functional/Teradata/TeradataBaseTestCase.php ? protoze nikde to nemuzu najit, ze by se takova tabulka nakonec pouzila

Reply:
to je ta z toho odstranenyho docasnyho testu ([tady](https://github.com/keboola/php-db-import-export/pull/165/commits/24e22d032f02bf854c5b42d6a0e21a562c5bf0f9)) - muze se smazat